docs/library/network.WLAN.rst

```diff
@@ -85,7 +85,17 @@ Methods
         * ``STAT_GOT_IP`` -- connection successful.
 
     When called with one argument *param* should be a string naming the status
-    parameter to retrieve.  Supported parameters in WiFI STA mode are: ``'rssi'``.
+    parameter to retrieve, and different parameters are supported depending on the
+    mode the WiFi is in.
+
+    In STA mode, passing ``'rssi'`` returns a signal strength indicator value, whose
+    format varies depending on the port (currently supported on ESP32 and ESP8266).
```

Comment:
rssi is actually supported on all WLAN interfaces except the cc3200 port

Reply:
Interesting, I've searched for `MP_QSTR_rssi` and only found mentions on ESP32 and ESP8266.  `rssi` appeared on CC3200 but from a quick look it didn't seem like it was handled, I guess I got it wrong.  I'll update the documentation for this and `stations` below.  Thanks!